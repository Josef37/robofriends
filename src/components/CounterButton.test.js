import React from "react";
import { shallow, mount } from "enzyme";
import CouterButton from "./CounterButton";

test("renders component for the first time right", () => {
  const wrapper = shallow(<CouterButton />);
  expect(wrapper.debug()).toMatchSnapshot();
});

test("counts after clicking the counter", () => {
  const wrapper = mount(<CouterButton />);
  const button = wrapper.find("button");
  button.simulate("click");
  expect(wrapper.debug()).toMatchSnapshot();
});
