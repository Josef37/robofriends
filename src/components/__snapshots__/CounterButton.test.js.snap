// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`counts after clicking the counter 1`] = `
"<CounterButton>
  <button onClick={[Function: onClick]}>
    Count: 
    1
  </button>
</CounterButton>"
`;

exports[`renders component for the first time right 1`] = `
"<button onClick={[Function: onClick]}>
  Count: 
  0
</button>"
`;
